### ISSUE TITLE
KIF with SwiftUI element

### ISSUE THREAD

**Issue by blyscuit** [kif-framework/KIF#1270]
My SwiftUI view is a simple view

```
struct ContentView: View {

    var body: some View {
        Button {
        } label: {
            Text("Close")
        }
        .accessibilityIdentifier("button")
    }
}
```

KIF doesn't seem to be able to locate the button (or any other view types) with the accessibility identifier.

I followed the steps in README, not sure what is missing or if it's `UIAccessibilityElement *element = [self accessibilityElementWithLabel:label value:value traits:traits fromRootView:fromView error:error];` not being able to pick up SwiftUI view?

<img width="828" alt="Screen Shot 2022-12-02 at 14 55 15" src="https://user-images.githubusercontent.com/6356137/205245826-79a80cba-3e07-4bd6-98a0-f93eb77fd161.png">

[Full source code ](https://github.com/blyscuit/KIF-Quick-Nimble)

**Comment by dostrander** [comment]
I think it's becuase you are adding it to a UI test target, try doing this in your project in the other testing target and this should work

**Comment by blyscuit** [comment]
@dostrander  Thank you. Works great.
